examples/pydantic_ai_examples/chat_app.py

```diff
@@ -89,11 +89,12 @@ def to_chat_message(m: ModelMessage) -> ChatMessage:
     first_part = m.parts[0]
     if isinstance(m, ModelRequest):
         if isinstance(first_part, UserPromptPart):
-            return {
-                'role': 'user',
-                'timestamp': first_part.timestamp.isoformat(),
-                'content': first_part.content,
-            }
+            if isinstance(first_part.content, str):
+                return {
+                    'role': 'user',
+                    'timestamp': first_part.timestamp.isoformat(),
+                    'content': first_part.content,
+                }
```

Comment:
Can we handle this more thoroughly here? I'm not super familiar with the example but basically if we are going to show something here, feels like we should show how to handle other input kinds. But if this is really basically internal code then maybe it's fine as-is.